torchdynamo/skipfiles.py

```diff
@@ -154,6 +162,8 @@ def check(filename, allow_torch=False):
         return True
     if filename in FILENAME_ALLOWLIST:
         return False
+    if filename in lazy_allowlist():
```

Comment:
Perhaps refactor the existing list to be lazy + cached rather than adding a second list.

Reply:
sg